frappe/public/js/frappe/utils/common.js

```diff
@@ -274,6 +274,10 @@ frappe.is_mobile = function () {
 	return $(document).width() < 768;
 };
 
+frappe.is_large_screen = function () {
+	return $(document).width() > 1920;
```

Comment:
Width or height? :thinking:

Reply:
Specifically for this problem, height is more important.

Reply:
Yeah just realized it should be height